src/test/java/ch/wisv/areafiftylan/TicketRestIntegrationTest.java

```diff
@@ -68,517 +54,493 @@
-                filter(sessionFilter).
-                header(login.getCsrfHeader()).
-                when().
-                content(ticketReceiver.getUsername()).contentType(ContentType.TEXT).
-                post(TRANSFER_ENDPOINT + "/" + ticket.getId()).
-                then().
-                extract().response();
-
-        logout();
-
-        return r;
-    }
-
-    private TicketTransferToken addTicketTransferGetToken() {
-        String token = addTicketTransfer(user.getUsername(), userCleartextPassword).
-                then().
-                extract().path("object");
-
-        return tttRepository.findByToken(token).orElseThrow(() -> new TokenNotFoundException(token));
+        Assert.assertTrue(ticket.getOwner().equals(ticketOwner));
     }
 
     @Test
     public void testGetTicketInControlNoTickets() {
-        SessionData login = login(user.getUsername(), userCleartextPassword);
-
-        ticketRepository.delete(ticket);
+        User user = createUser();
 
+        //@formatter:off
         given().
-                filter(sessionFilter).
-                header(login.getCsrfHeader()).
-                when().
-                get(TICKETS_ENDPOINT + "/teammembers").
-                then().
-                statusCode(HttpStatus.SC_OK).
-                body("$", hasSize(0));
+            header(getXAuthTokenHeaderForUser(user)).
+        when().
+            get(TICKETS_ENDPOINT + "/teammembers").
+        then().
+            statusCode(HttpStatus.SC_OK).
+            body("$", hasSize(0));
+        //@formatter:on
     }
 
     @Test
     public void testGetTicketInControlOwnTicket() {
-        SessionData login = login(user.getUsername(), userCleartextPassword);
+        User user = createUser();
+        createTicketForUser(user);
 
+        //@formatter:off
         given().
-                filter(sessionFilter).
-                header(login.getCsrfHeader()).
-                when().
-                get(TICKETS_ENDPOINT + "/teammembers").
-                then().
-                statusCode(HttpStatus.SC_OK).
-                body("$", hasSize(1)).
-                body("owner.username", containsInAnyOrder(user.getUsername()));
+            header(getXAuthTokenHeaderForUser(user)).
+        when().
+            get(TICKETS_ENDPOINT + "/teammembers").
+        then().
+            statusCode(HttpStatus.SC_OK).
+            body("$", hasSize(1)).
+            body("owner.username", containsInAnyOrder(user.getUsername()));
+        //@formatter:on
     }
 
     @Test
     public void testGetTicketInControlTeamMemberTicket() {
-        User teamMate = createTeamReturnSingleMember();
+        User user = createUser();
+        createTicketForUser(user);
+        User teammate = createUser();
+        createTicketForUser(teammate);
 
-        SessionData login = login(user.getUsername(), userCleartextPassword);
+        Team team = createTeamWithCaptain(user);
+        team.addMember(teammate);
```

Comment:
This is not persisted in the database, which is why the request only has a team with one member.